engine/admin/index.md

```diff
@@ -159,7 +159,7 @@ you installed Docker, you may already have this file.
 7. Verify that the `docker` daemon is running as specified with the `ps` command.
 
     ```bash
-    $ ps aux | grep docker | grep -v grep
+    $ ps aux | grep [d]ocker
```

Comment:
The trick makes sense, but I think that documentation (and code samples) should be explicit even if it is a bit more verbose. This minimizes the prior knowledge you need to have before starting to learn Docker.

So I think we should keep the snippet as it is today, but I'll defer to @mstanleyjones on this.